worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/processor/heightmap/HeightmapProcessor.java

```diff
@@ -6,26 +6,27 @@
 import com.fastasyncworldedit.core.queue.IChunk;
 import com.fastasyncworldedit.core.queue.IChunkGet;
 import com.fastasyncworldedit.core.queue.IChunkSet;
+import com.fastasyncworldedit.core.queue.implementation.blocks.IntGetBlocks;
 import com.sk89q.worldedit.extent.Extent;
 import com.sk89q.worldedit.world.block.BlockState;
 import com.sk89q.worldedit.world.block.BlockTypesCache;
 
 import javax.annotation.Nullable;
 import java.util.Arrays;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.Future;
 
 public class HeightmapProcessor implements IBatchProcessor {
 
     private static final HeightMapType[] TYPES = HeightMapType.values();
     private static final int BLOCKS_PER_Y_SHIFT = 8; // log2(256)
     private static final int BLOCKS_PER_Y = 256; // 16 x 16
     private static final boolean[] COMPLETE = new boolean[BLOCKS_PER_Y];
-    private static final char[] AIR_LAYER = new char[4096];
+    private static final char[] AIR_LAYER_CHAR = new char[4096];
+    private static final int[] AIR_LAYER_INT = new int[4096];
```

Comment:
It's probably worth moving these to FaweCache as it could be useful (and likely used) elsewhere

Reply:
I'll most likely look at moving these fields to the ```FaweCache``` class once I get this particular pull request merged and am 100% sure moving those won't cause a negative impact on anything.